### PR TITLE
First ui pass

### LINKED ISSUE #10: grey box + search within friends should not show before there are friends to list

### PR DESCRIPTION
**Pull request by icbat**
I'm pretty happy with this as a first pass. I've manually tested out my changes.

One potentially bigger but subtle change was re-ordering the CSS includes on index.html so as to not get overridden by bootstrap.

I must say I kind of hate how much of the display/layout is all in that one file (letsPlay.js)

Fixes #10

### CODE REVIEW

**Comment by icbat** [comment]
Maybe this weekend @theTechnoWeenie -sempai will notice me!

File: src/static/css/styles.css [hunk 1]
```diff
@@ -22,24 +27,21 @@
   min-width:0px;
 }
 .steamify{
-  color:#FFF;
-  background:linear-gradient(#CCC, #000);
-  border-radius: 15px;
-  border-style:outset;
-  border-width:4px;
```
Comment (theTechnoWeenie) [inline src/static/css/styles.css:29]: BOOOO HIISSSSSS

**Comment by theTechnoWeenie** [comment]
The scroll bar is caused by `overflow:auto;`. Remove that, and we'll :shipit: !

**Comment by icbat** [comment]
Was lazy and did it on the wobsite, but it should be fixed now

**Pull request merged**